Modelica/Resources/ReleaseNotes/Modelica.UsersGuide.ReleaseNotes.Version_1_5.html

```diff
@@ -332,10 +332,10 @@ <h2><a name="info" id="info">Information</a></h2>
 <li>Upper case constants changed to lower case (cf.
 Modelica.Constants)</li>
 <li>Introduced Modelica.SIunits.Wavelength due to typo
-"Wavelenght"</li>
+"Wavelength"</li>
 <li>Introduced
 ModelicaAdditions.Blocks.Logical.Interfaces.Comparison due to typo
-"Comparision"</li>
+"Comparison"</li>
```

Comment:
Those two matches will be fixed anyway when the release notes are regenerated with your help.

Reply:
Oh well

Reply:
Actually I don't think we are readding the old release notes.

Reply:
Simply remove the old typo then (as you did for the source).

Reply:
well it is now.